modules/drivers/lidar_surestar/BUILD

```diff
@@ -0,0 +1,32 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "libsurestar_driver_component.so",
+    linkopts = ["-shared"],
+    linkstatic = False,
+    deps = [":driver"],
+)
+
+cc_library(
+    name = "driver",
+    srcs = glob(
+        ["src/**/*.cpp"]
+    ),
+    hdrs = glob(
+        ["include/**/*.h"]
```

Comment:
Try to fine grain Bazel build target. Follow Bazel Best Practices [here](https://docs.bazel.build/versions/master/bazel-and-cpp.html#best-practices).

Reply:
BUILD is refined accordingly